packages/subproviders/src/globals.d.ts

```diff
@@ -3,6 +3,59 @@
 // tslint:disable:async-suffix
 // tslint:disable:completed-docs
 
+// Ethereumjs-tx declarations
+
+// Ledgerco declarations
+interface ECSignatureString {
+    v: string;
+    r: string;
+    s: string;
+}
+interface ECSignature {
+    v: number;
+    r: string;
+    s: string;
+}
+interface ECSignatureBuffer {
+    v: number;
+    r: Buffer;
+    s: Buffer;
+}
```

Comment:
We should remove these types from `global.d.ts`? `ECSignatureBuffer` is already added to `@0xproject/types`, `ECSignature` as well. Let's use them from there.